lib/uuid_map.rb

```diff
@@ -36,6 +36,13 @@ def rewrite(data)
     end
   end
 
+  def remap(from, to)
+    if uuid_for(from) && !uuid_for(to)
+      mapping[to] = mapping.delete(from)
+      rewrite(mapping)
+    end
```

Comment:
I'm not so sure it's sensible for this to just silently fail if you pass it invalid data…